metricbeat/module/windows/perfmon/reader.go

```diff
@@ -170,6 +169,7 @@ func randSeq(n int) string {
 
 // Close will close the PDH query for now.
 func (re *Reader) Close() error {
+	windows.CloseHandle(re.event)
```

Comment:
The query should be closed before the event. Maybe call CloseHandle inside a defer statement?